src/main/groovy/app/DocGen.groovy

```diff
@@ -189,12 +190,23 @@ ${data.metadata.header[1]}"""])
         // Execute a command in the shell
         static private def Map shell(List<String> cmd) {
             def proc = cmd.execute()
-            proc.waitFor()
+            // for some VERY complex docs - the process implementation hangs on wait() ...
+            // switching to the below - seem to work but gets a weird NPE... 
+            // java.lang.NullPointerException: Cannot invoke method call() on null object
+            // at app.DocGen$Util.shell(DocGen.groovy:193)
+            ByteArrayOutputStream bosOut = new ByteArrayOutputStream()
```

Comment:
prefered to use ```NullOutputStream``` and ```NullInputStream```, using ```ByteArrayOutputStream``` is using inecesary memory and can even cause OOM

Reply:
which means you can't get ANY error message, whatsoever .. we have tested this with >2000 page tests ... any your get a 300 bytes ...